cwms-data-api/src/main/java/cwms/cda/api/project/ProjectPublishStatusUpdate.java

```diff
@@ -0,0 +1,130 @@
+import org.jetbrains.annotations.NotNull;
+
+public class ProjectPublishStatusUpdate implements Handler {
+    public static final String TAG = ProjectController.TAG;
+    public static final String PATH = "/projects/status-update";
+    private final MetricRegistry metrics;
+
+    private Timer.Context markAndTime(String subject) {
+        return Controllers.markAndTime(metrics, getClass().getName(), subject);
+    }
+
+    public ProjectPublishStatusUpdate(MetricRegistry metrics) {
+        this.metrics = metrics;
+    }
+
+    @OpenApi(
+            description = "Publishes a message on the office's STATUS queue that a project has "
+                    + "been updated for a specified application",
+            queryParams = {
+                @OpenApiParam(name = OFFICE, required = true, description = "The office "
+                        + "generating the message (and owning the project)."),
+                @OpenApiParam(name = PROJECT_ID, required = true, description = "The location "
+                        + "identifier of the project that has been updated"),
+                @OpenApiParam(name = APPLICATION_ID, required = true, description = "A text "
+                        + "string identifying the application for which the update applies."),
+                @OpenApiParam(name = SOURCE_ID, description = "An application-defined string "
+                        + "of the instance and/or component that generated the message. "
+                        + "If NULL or not specified, the generated message will not include "
+                        + "this item."),
+                @OpenApiParam(name = TIMESERIES_ID, description = "A time series identifier of "
+                        + "the time series associated with the update. If NULL or not "
+                        + "specified, the generated message will not include this item."),
+                @OpenApiParam(name = BEGIN, description = "The start time of the updates to "
+                        + "the time series. If NULL or not specified, the generated message "
+                        + "will not include this item."),
+                @OpenApiParam(name = END, description = "The end time of the updates to the "
+                        + "time series. If NULL or not specified, the generated message will "
+                        + "not include this item.")
+            },
+            method = HttpMethod.POST,
+            responses = {
+                @OpenApiResponse(status = STATUS_200, content = {
+                    @OpenApiContent(type = Formats.JSON, from = PublishStatusUpdateResult.class)}
+                    )},
+            tags = {TAG},
+            path = PATH
+    )
+    @Override
+    public void handle(@NotNull Context ctx) throws Exception {
+
+        try (final Timer.Context ignored = markAndTime("publish")) {
+            ProjectDao prjDao = new ProjectDao(JooqDao.getDslContext(ctx));
+            String office = requiredParam(ctx, OFFICE);
```

Comment:
I'll move projectId into the path.  When I wrote it I was thinking of it as a utility that didn't belong to anything but keying off the project makes sense.

Reply:
oh, it might not be typical but I think its fine.  The method is just a trigger.  post makes it clear that it has side effects and should prevent proxies and caches from messing with it.

Reply:
makes sense to me!